pybossa/util.py

```diff
@@ -1046,7 +1046,8 @@ def validate_required_fields(data):
         field_val = field_info.get('val')
         check_val = field_info.get('check_val')
         int_val = field_info.get('require_int')
-        import_data = data.get(field_name)
+        field_names_lower = {field_name.lower():v for k,v in data.items()}
```

Comment:
this can be outside of for loop.

Reply:
`field_name` is referenced from within the for-loop. I'm not sure if this can be moved outside of the loop. Could you show an example? Thanks!